src/editor/components/monaco-tools/delim-line-suggestion.js

```diff
@@ -0,0 +1,40 @@
+import * as monaco from "monaco-editor/esm/vs/editor/editor.api";
+
+import { makeSuggestionList } from "./make-suggestion-list";
+
+const { Keyword } = monaco.languages.CompletionItemKind;
+
+// FIXME validChunkFlags should probably move to state eventually, since
+// we've discussed allowing plugins to define flags that might only
+// be available at run time. but for now, we can just hardcode this here.
+export const validChunkFlags = ["skipRunAll"];
+
+export const delimLineSuggestion = (lineSoFar, knownChunkTypes, lineNumber) => {
```

Comment:
Would be nice to have a jest test for this function.

Reply:
added tests! :-)